ui/screens.py

```diff
@@ -137,7 +145,7 @@ def showEvent(self, e):
     def closeEvent(self, e):
         self.hide()
         e.ignore()
-    def stack(self, w):
+    def stack_(self, w):
```

Comment:
Trying to connect any common UI-modifying functions as Qt slots to simplify threaded operations.